<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Flipbook" failures="0" tests="5" name="testLayer.LoginTest" time="5.405" errors="1" timestamp="2022-11-25T13:33:03 EST" skipped="4">
  <testcase classname="testLayer.LoginTest" name="initsetup" time="5.405">
    <error message="Parameter &#039;srcDir&#039; is not a directory: &#039;C:\Users\Meet\AppData\Local\Temp\screenshot12418481125132877311.png&#039;" type="java.lang.IllegalArgumentException">
      <![CDATA[java.lang.IllegalArgumentException: Parameter 'srcDir' is not a directory: 'C:\Users\Meet\AppData\Local\Temp\screenshot12418481125132877311.png'
at org.apache.commons.io.FileUtils.requireDirectory(FileUtils.java:2635)
at org.apache.commons.io.FileUtils.copyDirectory(FileUtils.java:682)
at org.apache.commons.io.FileUtils.copyDirectory(FileUtils.java:630)
at org.apache.commons.io.FileUtils.copyDirectory(FileUtils.java:531)
at org.apache.commons.io.FileUtils.copyDirectory(FileUtils.java:502)
at BasePackage.BaseHRMclass.screenshots(BaseHRMclass.java:66)
at testLayer.LoginTest.initsetup(LoginTest.java:25)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:65)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:381)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:319)
at org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:803)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:589)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- initsetup -->
  <system-out/>
  <testcase classname="testLayer.LoginTest" name="Title" time="0.000">
    <skipped/>
  </testcase> <!-- Title -->
  <system-out/>
  <testcase classname="testLayer.LoginTest" name="Login" time="0.000">
    <skipped/>
  </testcase> <!-- Login -->
  <system-out/>
  <testcase classname="testLayer.LoginTest" name="Login" time="0.000">
    <skipped/>
  </testcase> <!-- Login -->
  <system-out/>
  <testcase classname="testLayer.LoginTest" name="Login" time="0.000">
    <skipped/>
  </testcase> <!-- Login -->
  <system-out/>
</testsuite> <!-- testLayer.LoginTest -->
